egs/ami/s5/run_sdm_lstm.sh

```diff
@@ -168,13 +168,26 @@ fi
 if [ $stage -le 14 ]; then
   local/nnet3/run_lstm.sh \
     --mic $mic \
-    --train-stage -5 \
+    --train-stage -1 \
     --speed-perturb true \
     --stage 7 \
-    --common-egs-dir exp/sdm1/nnet3/lstm_sp/egs \
+    --affix lc22 \
+    --norm-based-clipping false \
+    --clipping-threshold 20\
```

Comment:
this is the top level (run.sh level) file  so I don't take arguments from command line and directly set them in the script.